examples/md-flexible/CMakeLists.txt

```diff
@@ -4,29 +4,30 @@ file(
         "src/*.cpp"
         "src/*.h"
 )
+add_executable(md-flexible ${MDFlex_SRC})
+
+target_include_directories(md-flexible PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-if (AUTOPAS_INTERNODE_TUNING)
-    message(STATUS "AUTOPAS_INTERNODE_TUNING set, searching for mpi to link against md-flexible ...")
+include(autopas_yaml-cpp)
+
+option(AUTOPAS_INCLUDE_MPI "Enable this variable to use MPI." OFF)
```

Comment:
In this case we would need to activate MD_FLEXIBLE_USE_MPI and AUTOPAS_INCLUDE_MPI because the latter is used by the WrapMPI header. This header is not part of MD-Flexible's context which is why I don't want to use MD_FLEXIBLE_USE_MPI in this header. 
And requiring two flags to enable MPI will definitely cause confusion.

Reply:
> And requiring two flags to enable MPI will definitely cause confusion.

Agreed. So we just leave it as is. However, in that case, maybe add something to the md-flex readme that explains this.